components/connectionmanager/src/lib.rs

```diff
@@ -203,4 +213,33 @@ impl ConnectionManager {
         }
         join_all(futures).await;
     }
+
+    fn dns_seed(self: &Arc<Self>, mut min_addresses_to_fetch: usize) {
+        let mut dns_seeders = self.dns_seeders.to_vec();
+        dns_seeders.shuffle(&mut thread_rng());
+
+        for seeder in dns_seeders {
+            info!("Querying DNS seeder {}", seeder);
+            let addrs = match format!("{}:53", seeder).to_socket_addrs() {
```

Comment:
FYI `(seeder, 53).to_socket_addrs()` should also work, not sure if `53` is the right port to put here BTW